packages/devtools_app/lib/src/screens/vm_developer/vm_function_display.dart

```diff
@@ -78,30 +81,23 @@ class VmFuncDisplay extends StatelessWidget {
       return 'Unrecognized function kind: ${function.obj.kind}';
     }
 
-    final camelCase = RegExp(r'(?<=[a-z])[A-Z]');
-
     final kind = StringBuffer();
 
+    final addSpace = () => kind.write(kind.isNotEmpty ? ' ' : '');
```

Comment:
define a function here instead of using a closure. That would make this a bit more clear.

```
void addSpace() {
  kind.write(kind.isNotEmpty ? ' ' : '');
}
```

dart allows you to define functions within the scope of other functions

Reply:
Done